DREAM3DReviewFilters/util/ITKTransformHelpers.h

```diff
@@ -0,0 +1,194 @@
+#ifndef _itktransformhelpers_h_
+#define _itktransformhelpers_h_
+
+#include "H5Support/H5Lite.h"
+#include "H5Support/H5ScopedSentinel.h"
+#include "H5Support/H5Utilities.h"
+
+#include "SIMPLib/Geometry/TransformContainer.h"
+#include "SIMPLib/ITK/itkInPlaceDream3DDataToImageFilter.h"
+#include "SIMPLib/ITK/itkInPlaceImageToDream3DDataFilter.h"
+
+struct ITKTransformHelpers
+{
+  typedef itk::Dream3DImage<double, 2> ImageType;
+
+  ITKTransformHelpers(hid_t fileID, std::string groupName)
```

Comment:
ITKTransformHelpers(hid_t fileID, const std::string& groupName)

Pass the strings by const reference